src/populateExchange.test.ts

```diff
@@ -34,8 +34,8 @@ const schemaDef = `
   union UnionType = User | Todo
 
   type Query {
-    todos: [Todo]
-    users: [User]
+    todos: [Todo!]
```

Comment:
Added some nullablity to the schema for broader test coverage. This should also test the cases fix by #150